lumen/ai/agents.py

```diff
@@ -277,7 +277,9 @@ class ChatAgent(Agent):
         statistics about the data, and continuing the conversation.
 
         Is capable of providing suggestions to get started or comment on interesting tidbits.
-        If data is available, it can also talk about the data itself.""")
+        If data is available, it can also talk about the data itself.
```

Comment:
nit: the original sentence is a weak statement, but it is stronger if you switch the clauses separated by the comma

```suggestion
        It can talk about the data, if available.
```